README.md

```diff
@@ -87,7 +87,7 @@ DESCRIPTION
 EXAMPLES
   $ FLAGSMITH_ENVIRONMENT=x flagsmith get
 
-  $ flagsmith get <ENVIRONMENT_ID>
+  $ flagsmith get <ENVIRONMENT_API_KEY>
```

Comment:
It feels to me like these should just be 2 separate functions in the cli? Its odd to me that providing a different key gets me a different result.

Reply:
Bear in mind that the server key will work for getting the flag values, but the client key will not work for getting the environment document.